src/libopensc/card-muscle.c

```diff
@@ -18,9 +18,7 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
-#if HAVE_CONFIG_H
 #include "config.h"
-#endif
```

Comment:
The deletion above is actually not desired.

Please rebase your changes on the current master and apply your changes. This would also remove the merge commits.